daal4py/sklearn/linear_model/_linear_0_21.py

```diff
@@ -19,8 +19,9 @@
 from scipy import sparse as sp
```

Comment:
For review you just one file to make a change. For make it easier to leave comments and you did not have to forward each change to each file - 0.21, 0.22, 0.23. At the final stage, you simply copy the changes and carefully discard them into the backward files.